backend/data/road_segmenting/calculate_distance.py

```diff
@@ -0,0 +1,74 @@
+from math import *
```

Comment:
I'm sure Travis is already complaining about this, but [don't do this](https://stackoverflow.com/questions/2386714/why-is-import-bad?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa).
Alternatives would be `import math`, so you can call `ceil(x)` with `math.ceil(x)`
You can also shorten it (even though math is plenty short) by writing `import math as m`, which would let you run `m.ceil(x)`, though in this case I'd just use math without an alias.